### DISCUSSION TITLE
xdrip+OOP2 reading 6m

### DISCUSSION THREAD

**Discussion by idalioreis** [NightscoutFoundation/xDrip#2594] [General]
hi, I'm a libre user with xdrip and oop2, I have a doubt regarding the sensor reading time, because there is some conflict with aaps due to that time.
oop2 is reading every 2m then it reaches the xdrip with 6 minutes. aaps is prepared to receive 5m and receiving at 6m creates some problems.
is it normal to receive in 6m?

Thanks
![image](https://user-images.githubusercontent.com/48919798/210171433-f6997d2b-a83e-4498-9545-59ec2f4f18ed.png)
![image](https://user-images.githubusercontent.com/48919798/210171439-61add06f-3825-4382-b3da-dfb230b838fa.png)

**Comment by Navid200** [comment]
@psonnera

**Comment by PhilTU** [comment]
It should receive a value every minute, not every 2. The 5 minute Onward transmission will struggle as 2 into 5 doesn't go.

**Comment by tzachi-dar** [comment]
This might happen if the connection keeps terminating. It can only connect every 2 minutes.
But I find it hard to believe that it is never able to keep the connection for at least a minute.

We will need more logs. Do you see more information if you also turn on el and eh?

**Reply by idalioreis** [reply]
![Screenshot_2023-03-04-17-58-43-949_info nightscout androidaps](https://user-images.githubusercontent.com/48919798/222921656-9d050ec6-76c1-4ff3-b5db-5c4b06e3cb5b.jpg)
Change now to dexcom and the time between reading came normal, 5m so I think there is a bug. 
I use xdrip with dexcom. 
What log do you want?

**Reply by Navid200** [reply]
>  What log do you want?

You have posted xDrip logs already in your first post.   But, you have disabled EL and EH in it.

**Reply by idalioreis** [reply]
Can't understand what El and EH are?

**Reply by Navid200** [reply]
> Can't understand what El and EH are?

![210171439-61add06f-3825-4382-b3da-dfb230b838fa](https://user-images.githubusercontent.com/51497406/222923537-36915991-83cf-4abd-8390-e3da337dc557.png)
